src/ceph-disk/ceph_disk/main.py

```diff
@@ -122,7 +122,7 @@ def get_ready_by_type(what):
 
     @staticmethod
     def get_ready_by_name(name):
-        return [x[name]['ready'] for x in PTYPE.values()]
+        return [x[name]['ready'] for x in PTYPE.values() if x.get(name, None)]
```

Comment:
makes sense... but i am not sure i understand ceph-disk enough to comment on this. @dachary what do you think?